spec/controllers/questionnaire_controller_spec.rb

```diff
@@ -121,6 +121,56 @@
         Timecop.return
       end
     end
+
```

Comment:
not sure if we have a swagger spec for this route, but if we do also update that one.

Reply:
Thanks, I'll run the swagger thing. We should really have a circleCI job that checks if the swagger spec is up-to-date.

Reply:
Fixed.